components/lib/crash/src/main/java/mozilla/components/lib/crash/db/ReportEntity.kt

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.lib.crash.db
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import mozilla.components.lib.crash.service.CrashReporterService
+
+/**
+ * Datanase entry describing a crash report that was sent to a crash reporting service.
+ */
+@Entity(
+    tableName = "reports"
+)
+internal data class ReportEntity(
+    /**
+     * Database internal primary key of the entry.
+     */
+    @PrimaryKey(autoGenerate = true)
+    @ColumnInfo(name = "id")
+    var id: Long? = null,
+
+    /**
+     * UUID of the crash that was reported.
+     */
```

Comment:
nit: should we name this column as well to follow the format of the other columns?

Reply:
Yeah, makes sense. I'll update the PR.